### PR TITLE
Workaround for #214

### LINKED ISSUE #214: Dynamic require() breaks RN 49.0+
screen shot 2017-10-05 at 4 31 47 pm

This error appears when trying to run our app with react native 0.49. It appears to be breaking a few other libraries including realm and moment.js (https://github.com/facebook/react-native/issues/16216)

### PR DESCRIPTION
**Pull request by danieloprado**
Fix #214 

…and proper workaround/fix for proposed changes in metro bundler https://github.com/facebook/metro-bundler/pull/69

### CODE REVIEW

File: src/lang.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ var Messages = require('./messages');
 
 require('./lang/en');
 
+var require_method = require;
```
Comment (iamdtang) [inline src/lang.js:5]: Could you make this camelCased to be consistent with the rest of the code?

File: src/lang.js [hunk 2]
```diff
@@ -42,8 +44,10 @@ var container = {
    */
   _load: function(lang) {
     if (!this.messages[lang]) {
-      var rawMessages = require('./lang/' + lang);
-      this._set(lang, rawMessages);
+      try {
+        var rawMessages = require_method('./lang/' + lang);
+        this._set(lang, rawMessages);
+      } catch (e) {}
```
Comment (iamdtang) [inline src/lang.js:50]: How about throwing an error like "Could not require language {lang}"?

**Review by iamdtang — COMMENTED** [review]
Looks fine to me. Just a couple of small changes. What do you think @mikeerickson? Thanks!

File: src/lang.js (hunk 1 above)
Comment (mikeerickson) [reply src/lang.js:5]: I am fine with it @skaterdav85 but do agree this should be camelCased

**Comment by agrosner** [comment]
do you have an idea when this will get merged and new version?

**Pull request merged**